packages/sku/scripts/init.js

```diff
@@ -148,7 +148,7 @@ const getTemplateFileDestinationFromRoot =
   };
   const packageJsonString = JSON.stringify(packageJson, null, 2);
 
-  fs.writeFileSync(path.join(root, 'package.json'), packageJsonString);
+  fs.writeFile(path.join(root, 'package.json'), packageJsonString);
```

Comment:
Should this be awaited?

```suggestion
await  fs.writeFile(path.join(root, 'package.json'), packageJsonString);
```

Reply:
https://github.com/seek-oss/sku/pull/770/commits/1f137e4372bcab5dc0546fa6da91272f496da813